src/openrct2/network/network.h

```diff
@@ -52,6 +51,14 @@ extern "C" {
 
 #ifndef DISABLE_NETWORK
 
+#ifdef __cplusplus
+extern "C" {
+#endif // __cplusplus
+#include <openssl/evp.h>
```

Comment:
There is another set of includes just a few lines down from here, that is already guarded by `#ifdef __cplusplus`, can you move it there?